File: week6-Python/exercises/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

for i in range(n):
    for k in range(n - 1, i, -1):
        print(" ", end="")
    for j in range(i + 1):
        print("#", end="")
    for x in range(1):
        print("  ", end="")
    for z in range(i + 1):
        print("#", end="")

    print()

File: week1-C/exercises/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int get_height(void);

int main(void)
{
    int n = get_height();

    for (int i = 0; i < n; i++)
    {
        for (int k = n - 1; k > i; k--)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

int get_height(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    return n;
}
